packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,158 +24,215 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-border-radius-xs * 2;
 
-  @include bpk-border-radius-pill;
+  svg {
+    /* stylelint-disable declaration-no-important */
+    width: 16 * $bpk-one-pixel-rem !important;
+    height: 16 * $bpk-one-pixel-rem !important;
+  }
+
+  &__leading-accessory-view {
+    display: inline-flex;
+    fill: currentColor;
+
+    @include bpk-margin-trailing(bpk-spacing-md());
+  }
+
+  &__trailing-accessory-view {
+    display: inline-flex;
+    fill: currentColor;
+
+    @include bpk-margin-leading(bpk-spacing-md(), false);
+    @include bpk-margin-trailing(-1 * bpk-spacing-md(), false);
+  }
 
```

Comment:
Without the tail icon, the distance between the text and the border is 16px, and with the tail icon, the distance between the icon and the border is 8px.